docs/content/overview.md

```diff
@@ -24,6 +24,7 @@ Here is an overview of the major features of each cloud storage system.
 | Citrix ShareFile             | MD5               | R/W     | Yes              | No              | -         | -        |
 | Dropbox                      | DBHASH ¹          | R       | Yes              | No              | -         | -        |
 | Enterprise File Fabric       | -                 | R/W     | Yes              | No              | R/W       | -        |
+| Files.com                    | MD5, CRC32        | DR/W    | Yes              | No              | R         | -        |
```

Comment:
You haven't implemented setting directory mod times so this should just be `R/W`

It is relatively easy to do if you want to

Reply:
Added directory mod time setting.

Reply:
Actually, still working on this ...

Reply:
Updated.